Index: linux-4.14.77/arch/powerpc/platforms/85xx/Kconfig
===================================================================
--- linux-4.14.77.orig/arch/powerpc/platforms/85xx/Kconfig
+++ linux-4.14.77/arch/powerpc/platforms/85xx/Kconfig
@@ -60,6 +60,17 @@ config HIVEAP_330
       This board is a Concurrent Dual-Band wireless access point with a
       Freescale P1020 SoC.
 
+config PANDA
+	bool "OCEDO PANDA"
+	select DEFAULT_UIMAGE
+	select ARCH_REQUIRE_GPIOLIB
+	select GPIO_MPC8XXX
+	help
+	  This option enables support for the OCEDO PANDA board.
+
+	  This board is a Concurrent Dual-Band wireless access point with a
+	  Freescale P1020 SoC.
+
 config MPC8540_ADS
 	bool "Freescale MPC8540 ADS"
 	select DEFAULT_UIMAGE
Index: linux-4.14.77/arch/powerpc/platforms/85xx/Makefile
===================================================================
--- linux-4.14.77.orig/arch/powerpc/platforms/85xx/Makefile
+++ linux-4.14.77/arch/powerpc/platforms/85xx/Makefile
@@ -11,6 +11,7 @@ obj-$(CONFIG_BSC9131_RDB) += bsc913x_rdb
 obj-$(CONFIG_BSC9132_QDS) += bsc913x_qds.o
 obj-$(CONFIG_C293_PCIE)   += c293pcie.o
 obj-$(CONFIG_HIVEAP_330) += hiveap-330.o
+obj-$(CONFIG_PANDA)       += panda.o
 obj-$(CONFIG_MPC8540_ADS) += mpc85xx_ads.o
 obj-$(CONFIG_MPC8560_ADS) += mpc85xx_ads.o
 obj-$(CONFIG_MPC85xx_CDS) += mpc85xx_cds.o
